specviz/widgets/plotting.py

```diff
@@ -1,20 +1,25 @@
-import logging
 import os
 
 import astropy.units as u
 import numpy as np
+import logging
 import pyqtgraph as pg
 import qtawesome as qta
+
 from qtpy.QtCore import Property, QEvent, QModelIndex, QObject, Qt, Signal
 from qtpy.QtWidgets import (QAction, QListWidget, QMainWindow, QMdiSubWindow, QDialog,
-                            QToolButton, QSizePolicy, QWidget, QMenu, QWidgetAction, QColorDialog, QMessageBox)
+                            QToolButton, QSizePolicy, QWidget, QMenu, QWidgetAction, QColorDialog, QMessageBox, QDialogButtonBox)
 from qtpy.uic import loadUi
 
 from ..core.items import PlotDataItem
 from ..core.models import PlotProxyModel
 from ..utils import UI_PATH
 from .custom import LinearRegionItem
 
+logging.basicConfig(level=logging.DEBUG, format="%(filename)s: %(levelname)8s %(message)s")
```

Comment:
Are these different than those in #429?

Reply:
No, I'm removing them now

Reply:
I've noticed you used logging in the setter functions for data_unit and spectral_axis, should I just leave it in?